platform-api/analytics-api-core/src/main/scala/org/ekstep/analytics/api/service/DeviceRegisterService.scala

```diff
@@ -45,12 +46,17 @@ class DeviceRegisterService extends Actor {
         println(s"did: $did | device_spec: ${body.request.dspec} | uaspec: ${uaspec.getOrElse("")}")
         if (validIp.nonEmpty) {
             val location = resolveLocation(validIp)
-            println(s"Resolved Location for device_id $did: $location")
-            val channel = body.request.channel.getOrElse(defaultChannel)
+
+            // logging metrics
+            APILogger.log("", Option(Map("comments" -> s"$did resolved to state: ${location.state} city: ${location.city}")), "registerDevice")
```

Comment:
Move this out of "if" block so that we would know whether a did has been resolved or not